RFC-by-stage/1-approved/update-grid.md

```diff
@@ -0,0 +1,26 @@
+---
+Status: `Proposal` # Please do not change this.
+Implementer: # It will be changed upon merging and as it moves through the RFC stages
+---
+
+# Update Grid 12 to use Flexbox
+
+## The issue to be solved
+
+Grid 12 is how the Design System does layout.
+
+Grid 12 was based on Bootstrap 3 (I think, it might be v4) and uses the box model to handle layout. Bootstrap v3 Grid docs here https://getbootstrap.com/docs/3.3/css/#grid and v4 here https://getbootstrap.com/docs/4.0/layout/grid/.
+
+However, Bootstrap is now at v5.1 and has moved to Flexbox, see https://getbootstrap.com/docs/5.1/layout/grid/, which is a good signal that we should too.
+
+There are increased benefits of using Flexbox - such as easier vertical centering, which would add capability to Grid 12.
+
+A downside is that this would limit browser compatibility, however, Flexbox is highly supported by browsers nowadays, see https://caniuse.com/flexbox.
```

Comment:
Is our preference to ship modern unprefixed CSS code in the design system and leave it for implementers to use something like autoprefixer to add the vendor specific prefixes if they’re targeting older browsers, or a prefixed version, or both?

Reply:
That is a very good question, I don't know yet. I would lean to saying - "yes we should ship unprefixed", and then offer a single package export of the complete minified + autoprefixed. So for instance: 

`import ButtonStyles from "@gold.au/buttons/css/button.css";   // 👈  unprefixed`

vs 

`import GoldStyles from '@gold.au/design-system"; // 👈  prefixed + minified`

or 

`<link href=unpkg.com/@gold.au/design-system@16.7.0/gold-design-system.min.css />`

This said, I will open this now as a new RFC 👍

Reply:
Opening this https://github.com/designsystemau/RFCs/pull/16/files. So will resolve this conversation.